clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2212,6 +2212,39 @@ device kernel, the attribute is ignored and it is not propagated to a kernel.
   }];
 }
 
+def SYCLIntelStallEnableAttrDocs : Documentation {
+  let Category = DocCatFunction;
+  let Heading = "intel::stall_enable";
+  let Content = [{
+When applied to a lambda or function call operator (of a function object)
+on device, this requests, to the extent possible, that statically-scheduled
+clusters handle stalls using a stall-enable signal to freeze computation
+within the cluster.
```

Comment:
Do we need to say that this is ignored on host,  @Fznamznon?
If so, suggest, somethine like: 'This attribute is ignored on the host'.

Reply:
I don't have a strong opinion here, other FPGA attributes are likely also ignored on host, but most of their docs don't mention anything about host. This doc also says "When applied to a lambda or function call operator (of a function object) **on device** " and nothing about host, which might be a clue that this attribute is intended for device use. However explicit statement that this attribute doesn't have any effect on host does seem good. So, I think we could add it, but it is not really something that should block this patch if we don't do it.

Reply:
I think it is good for users to mention about this. I have updated doc.